api/config/hooks.go

```
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"log"
+
+	"gorm.io/gorm"
+)
+
+// SetupHooks configura los hooks personalizados para GORM
+func SetupHooks(db *gorm.DB) {
+	// Agregar callback para ignorar errores específicos de columnas
+	db.Callback().Create().Before("gorm:create").Register("app:handle_missing_columns", handleMissingColumns)
```

golangci-lint (1.64.8)

Error return value of `(*gorm.io/gorm.callback).Register` is not checked

(errcheck)